manifests/deploy/deployment.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: combo
```

Comment:
If we're going to be hardcoding the namespace with these manifests, we should introduce an explicit Namespace YAML manifest as well.

Reply:
Went ahead and added a `namespace.yaml`. Also updated the Makefile targets to create it before the controller.